src/main/java/karuna/karuna_backend/visitor/message/domain/VisitorMessageService.java

```diff
@@ -15,4 +20,17 @@ public VisitorMessageDto sendMessage(VisitorMessageCreateDto visitorMessageCreat
         VisitorMessage visitorMessage = VisitorMessageMapper.toModel(visitorMessageCreateDto);
         return VisitorMessageMapper.toDto(visitorMessageRepository.save(visitorMessage));
     }
+
+
+    public VisitorMessageDtoList getMessages(VisitorMessageRequest visitorMessageRequest) {
```

Comment:
VistiroMessageWrapper not will be better?

Reply:
i will change it